wagtail/contrib/postgres_search/backend.py

```diff
@@ -228,7 +228,9 @@ def get_search_field(self, field_lookup, fields=None):
                 return self.get_search_field(sub_field_name, field.fields)
 
     def prepare_word(self, word):
-        return unidecode(word)
+        # remove backslash from string to prevent syntax error in tsquery
+        stripped_word = word.replace("\\", "")
```

Comment:
This is removing an important character that can be separating words: we must escape backslashes, not remove them.
According to [the tsquery documentation](https://www.postgresql.org/docs/11/datatype-textsearch.html), backslashes must be doubled, though no example is available.
Also, a better place to put it would be in `RawSearchQuery`.
So please update your PR to remove this change and add new `.replace('\\', '\\\\')` at this line: https://github.com/wagtail/wagtail/blob/ddc5f1c21c74a1ea9a2ae819fe1dfce419655808/wagtail/contrib/postgres_search/models.py#L21

Reply:
Make sense. I've made that update.